### DISCUSSION TITLE
Columns Letter Case

### DISCUSSION THREAD

**Discussion by MichalisDBA** [osalvador/ReplicaDB#99] [General]
Hello.

Is there a parameter or something in the config of replicadb to remove the uppercase function when selecting columns of a table?

For example we have a source table

Table1 -> Columns -> ColumnTest1, ColumnTest2

When we run replicadb with source table

SELECT * FROM Table1

the return columns are COLUMNTEST1, COLUMNTEST2

ReplicaDB automatically converts the letter case to uppercase.

We have a problem at Oracle sink database that some tables have DDL with specified columns "ColumnTest1", "ColumnTest2"

I know it's a bad practise bad this is how it has been setup and now after trying to migrate data between MSSQL to Oracle it throws an error

ORA-00904: invalid identifier because replicadb tries to insert uppercase columns to lower case columns.

**Comment by osalvador** [comment]
Hi @MichalisDBA ,

Try this undocumented parameter: 

`--quoted-identifiers                                    Should all database identifiers be quoted.`

For example: 

```
~/replicadb/bin> ./replicadb --quoted-identifiers --options-file replicadb.conf
```

It is only documented in the help command: 
```
~/replicadb/bin> ./replicadb -h 
usage: replicadb [OPTIONS]

Arguments:
    --bandwidth-throttling <KB/s>                           Adds a bandwidth cap for the replication in KB/sec.
    --fetch-size <fetch-size>                               Number of entries to read from database at once.
 -h,--help                                                  Print this help screen
 -j,--jobs <n>                                              Use n jobs to replicate in parallel. Default 4
    --mode <mode>                                           Specifies the replication mode. The allowed values are complete,
                                                            complete-atomic, incremental or cdc.
    --options-file <file-path>                              Options file path location
    --quoted-identifiers                                    Should all database identifiers be quoted.
```

**Reply by MichalisDBA** [reply]
Yes it works.

Thank you very much!

**Reply by osalvador** [reply]
Great!
